requirements.txt

```diff
@@ -11,7 +11,7 @@ pandas
 pyyaml
 scipy
 tables
-# vivarium_inputs[data]>=5.0.0
+# vivarium_inputs>=5.0.0
```

Comment:
Why did this go away?

Reply:
It's never even been a thing. The only repos with a data_requirements extra defines are model repos